.github/ISSUE_TEMPLATE/rollout.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Rollout Request
+about: Create an issue to track the rollout of a new release
+title: ''
+labels: rollout
+assignees: ''
+
+---
+
+<!-- Please answer these questions when reporting a bug. Thanks! -->
+
+- [ ] Cut release v1.x.y-rc0 - ETA: 06/28/2021
+
+- [ ] Rollout v1.x.y-rc0 to testnet - ETA: 06/28/2021
+
+- [ ] Stress test v1.x.y-rc0 on testnet - ETA: 07/05/2021
+
+- [ ] Rollout v1.x.y-rc0 to canary API nodes on mainnet - ETA: 07/12/2021
+
```

Comment:
should we move this to after critical problem is fixed, and official v1.x.y is released?

critical problem could potential create a hard-fork in worst case

Reply:
Moved to the top